### PR TITLE
LLMonitor Callback Handler integration

### PR DESCRIPTION
**Pull request by vincelwt**
Hi!

This PR adds a Callback Handler for [LLMonitor](https://llmonitor.com), an open-source llm-ops platform.

This brings parity with the Python version where the callback handler [is already available](https://python.langchain.com/docs/integrations/callbacks/llmonitor). 

I added a new integration test as well as the relevant peer dependency and documentation.

The code was formatted and linted.

Twitter handles: `@llmonitor_hq` / `@vincelwt`

Let me know if anything needs to be changed.

Thank you and have a wonderful day.

### CODE REVIEW

File: langchain/scripts/create-entrypoints.js [hunk 1]
```diff
@@ -221,7 +222,7 @@ const entrypoints = {
   "storage/in_memory": "storage/in_memory",
   "storage/ioredis": "storage/ioredis",
   // hub
-  "hub": "hub",
```
Comment (jacoblee93) [inline langchain/scripts/create-entrypoints.js:224]: Mind reverting this?

**Comment by jacoblee93** [comment]
Hey @vincelwt made a few small lint/build changes - most visible one is that we currently enforce entrypoints to be the same as paths under `src`.

If you really want to keep it as `langchain/callbacks/llmonitor`, we could make an `index.ts` file that exports the handler. LMK what you want to do.

**Comment by vincelwt** [comment]
Hi @jacoblee93 thanks for reviewing, making the changes asap

**Comment by jacoblee93** [comment]
Ah @vincelwt already made them! Just wanted to make sure you saw and were aware of the entrypoint change

**Comment by vincelwt** [comment]
Oh awesome @jacoblee93, all good for me with the entrypoint change - thanks a lot

**Comment by jacoblee93** [comment]
Awesome! Will get this merged and in the next release.

**Pull request merged**